client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -1,18 +1,49 @@
-import { Grid, Paper } from '@material-ui/core';
+import { Grid, Paper, CircularProgress, Typography, Button } from '@material-ui/core';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
 
 import Sidebar from '../Sidebar/SideBar';
 import useStyles from './useStyles';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { useAuth } from '../../context/useAuthContext';
+import { useHistory } from 'react-router-dom';
 
 const ProfilePhoto = (): JSX.Element => {
   const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const history = useHistory();
+  const handleUpload = (event: any) => {
+    console.log(event.target.files[0]);
+  };
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    // loading for a split seconds until history.push works
+    return <CircularProgress />;
+  }
   return (
     <Grid container>
       <Grid item xs={2}>
         <Sidebar></Sidebar>
       </Grid>
       <Grid item xs={10}>
         <Paper className={classes.root} elevation={2}>
-          <h1>Profile Photo</h1>
+          <Grid container justify="center" className={classes.main}>
+            <Grid xs={6} container alignItems="center" direction="column">
```

Comment:
Grid containers don't accept `xs={6}` this would be causing an error in your code.